runhouse/servers/obj_store.py

```diff
@@ -603,3 +603,95 @@ def get_obj_refs_dict(self, d: Dict[Any, Any]):
         return {
             k: self.get(v, default=v) if isinstance(v, str) else v for k, v in d.items()
         }
+
+    ##############################################
+    # More specific helpers
+    ##############################################
+    def put_resource(
+        self,
+        serialized_data: Any,
+        serialization: Optional[str] = None,
+        env_name: Optional[str] = None,
+    ) -> "Response":
+        from runhouse.servers.http.http_utils import deserialize_data
+
+        if env_name is None:
+            if self.servlet_name is None:
+                raise ObjStoreError("No env name provided and no servlet name set.")
+            else:
+                env_name = self.servlet_name
```

Comment:
If we're inside this env, should we just call self.put_resource_local directly? No reason to serialize, deserialize, and add a ray call to the servlet if we're already in the right env process?

Reply:
fixed